front-end-side/src/Pages/Friends.js

```diff
@@ -1,10 +1,13 @@
 import React from "react";
 import Navbar from "../../src/Components/Navbar";
+import "../Pages/Friends.css";
 const Friends = () => {
     return (
         <div>
             <Navbar/>
-            <h1>Friends</h1>
+            <h1>Friends:</h1>
+            <h2>Regex</h2>
+            <img src="/dog.gif" alt=""/>
```

Comment:
Why alt tag is empty? What will will be shown if image will not be loaded

Reply:
Is the .gif optimized?

Reply:
Thanks for the "alt" reminder. Optimize the gif yourself.